webpack.common.js

```diff
@@ -79,10 +79,7 @@ module.exports = {
 			},
 			{
 				test: /\.(png|jpg|gif|svg)$/,
-				loader: 'file-loader',
-				options: {
-					name: '[name].[ext]?[hash]',
-				},
+				loader: 'url-loader',
```

Comment:
changes are copied from https://github.com/nextcloud/webpack-vue-config/blob/master/webpack.js

Reply:
You can't use the webpack-vue-config directly?

Reply:
well this needs backporting so lets do it step for step